tests/test_flop_count.py

```diff
@@ -112,6 +112,20 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
         return x
 
 
+class BMMNet(nn.Module):
+    """
+    A network with a single torch.bmm operation. This is used for testing
+    flop count for torch.bmm.
+    """
+
+    def __init__(self) -> None:
+        super(BMMNet, self).__init__()
+
```

Comment:
```suggestion
```

Reply:
Hi, remove the blank line(line-123) will raise the format check error.

Reply:
Oh I mean the init method can be removed

Reply:
Done. (By the way, the init method of MatmulNet can be removed also)